app/controllers/admin_shelters_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AdminSheltersController < ApplicationController
+  def index
+    @shelters = Shelter.find_by_sql("SELECT * FROM shelters ORDER BY name DESC")
```

Comment:
This would be considered an MVC infraction because you are doing SQL logic here in the controller. We would want to move this logic to the shelter model in some method. Then we would call that method here in the controller.